src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -311,6 +313,22 @@ private static void MutationFunctionsTestSetup(RecalcEngine engine, bool numberI
 
             var nameTableType = TableType.Empty().Add("name", FormulaType.String);
             engine.UpdateVariable("t_name", FormulaValue.NewTable(nameTableType.ToRecord()));
+
+            var r3Table = TableType.Empty().Add("b", numberType);
+            var r3Type = RecordType.Empty()
+                .Add(new NamedFormulaType("a", r3Table, "DisplayNamea"));

```

Comment:
>DisplayNamea [](http://example.com/codeflow?start=57&length=12)

Can you use the display name in the tests?

Reply:
added a test . is that enough ?